### PR TITLE
Send Mirror refreshed records in batches

### PR DESCRIPTION
**Pull request by gavin-norman-sociomantic**
Blocked until swarm v4.5.0 is released.

### CODE REVIEW

**Comment by gavin-norman-sociomantic** [comment]
Hmm, two of the builds (https://travis-ci.org/sociomantic-tsunami/dhtproto/jobs/319197891, https://travis-ci.org/sociomantic-tsunami/dhtproto/jobs/319197881) stalled in the dhtrestart test. I can't reproduce this locally, but will investigate.

**Comment by gavin-norman-sociomantic** [comment]
There are some suspicious looking ["small delays"](https://github.com/sociomantic-tsunami/dhtproto/blob/v13.x.x/test/dhtrestart/main.d#L70-L71) in the test that hangs. Pushed an attempted fix.

**Comment by gavin-norman-sociomantic** [comment]
All the other PRs on neo pass, so it must be something that this PR specifically changes that's breaking the tests.

**Comment by gavin-norman-sociomantic** [comment]
Interesting. Building at commit 2/4 with `F=production`, I can reproduce a hang locally. The test "Neo Puts followed by neo Mirror with connection drop" never completes. (Though note that this is _not_ the test that hangs in Travis.)

**Comment by gavin-norman-sociomantic** [comment]
Ok this is getting interesting now. Building locally with `F=production`, one of the tests hangs. If I disable `-O`, the hang doesn't happen.

**Comment by gavin-norman-sociomantic** [comment]
Ditto for `F=devel` plus `-O`.

**Comment by gavin-norman-sociomantic** [comment]
The problem has been discovered; see #88. The first two commits of this PR are now fixed and pass tests. Testing the full RP again now.

**Comment by gavin-norman-sociomantic** [comment]
Green! :tada:

File: src/dhtproto/node/neo/request/Mirror.d [hunk 1]
```diff
@@ -383,6 +393,10 @@ public abstract scope class MirrorProtocol_v0
         this.value_buffer = this.resources.getVoidBuffer();
         this.update_queue.initialise(*this.resources.getVoidBuffer());
         this.refresh_queue.initialise(*this.resources.getVoidBuffer());
+        const max_batch_size = 64 * 1024; // TODO: read from config
```
Comment (nemanja-boric-sociomantic) [inline src/dhtproto/node/neo/request/Mirror.d:396]: Double checking: this TODO should be merged at this stage, right?

**Review by nemanja-boric-sociomantic — APPROVED** [review]

File: src/dhtproto/node/neo/request/Mirror.d (hunk 1 above)
Comment (gavin-norman-sociomantic) [reply src/dhtproto/node/neo/request/Mirror.d:396]: Yeah, this is something that'll be addressed in the future. (#75)

**Pull request merged**